static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -294,8 +294,7 @@ const makeEpicABTestVariant = (
             return (
                 meetsMaxViewsConditions &&
                 matchesCountryGroups &&
-                matchesTags &&
-                matchesSections &&
+                matchesTagsOrSections &&
```

Comment:
Wouldn't this have been clearer if you used `&& (matchesTags || matchesSections) &&` ? Or maybe you definitely need the two lengths and'd?

Reply:
Perhaps there's a neater way to express this, but it also needs to allow for empty tags/sections lists (which means no filtering)